document_page_approval_levels/models/document_page.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+############################################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 206 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: Luis Torres <luis_t@vauxoo.com>
+#    planned by: Sabrina Romero <sabrina@vauxoo.com>
+############################################################################
+from openerp import models, fields, api
+
+
+class DocumentPage(models.Model):
+    """This class is use to manage Document."""
+    _inherit = 'document.page'
+
+    approver_items_id = fields.One2many(
```

Comment:
the field has a semantic issue:

s/approver_items_id/approver_item_ids/g

Set a proper help please to explain the meaning functionally.